src/solvers/lowering/byte_operators.cpp

```diff
@@ -2136,10 +2136,30 @@ static exprt lower_byte_update(
         instantiate_byte_array(value_as_byte_array, 0, (type_bits + 7) / 8, ns);
     }
 
+    const std::size_t update_size = update_bytes.size();
```

Comment:
While we're here: recommend always using unit suffixes in this code (update_size_bytes, width_bits)

Reply:
Done.